.github/workflows/testing.yml

```diff
@@ -60,7 +60,15 @@
        
        - name: spatial versions
          run: 'python -c "import geopandas; geopandas.show_versions();"'
-       
+
+       - name: declare doctest run
+         run: |
```

Comment:
I think this is fine but it may be worth adding either as a separate action (as in [momepy](https://github.com/pysal/momepy/blob/main/.github/workflows/test_docstrings.yaml)) or as a separate job (as in [geopandas](https://github.com/geopandas/geopandas/blob/ac87e9ebb3b8e7e3eb4632d64cc3c5a930625dbb/.github/workflows/tests.yaml#L107-L110))

Reply:
I'll add that as a To DO issue.